frontend/src/scenes/events/EventsTable.tsx

```diff
@@ -455,162 +477,176 @@ export function EventsTable({
+                />
+                <LemonTable
+                    data-attr={dataAttr}
+                    dataSource={eventsFormatted}
+                    loading={isLoading}
+                    columns={columns}
+                    key={selectedColumns === 'DEFAULT' ? 'default' : selectedColumns.join('-')}
+                    className="ph-no-capture"
+                    loadingSkeletonRows={20}
+                    emptyState={
+                        isLoading ? undefined : properties.some((filter) => Object.keys(filter).length) ||
+                          eventFilter ? (
+                            `No events matching filters found in the last ${months} months!`
+                        ) : (
+                            <>
+                                This project doesn't have any events. If you haven't integrated PostHog yet,{' '}
+                                <Link to="/project/settings">
+                                    click here to instrument analytics with PostHog in your product
+                                </Link>
+                                .
+                            </>
+                        )
+                    }
+                    rowKey={(row) =>
+                        row.event ? row.event.id + '-' + row.event.event : row.date_break?.toString() || ''
+                    }
+                    rowClassName={(row) => {
+                        return clsx({
+                            'event-row': row.event,
+                            highlighted: row.event && highlightEvents[row.event.id],
+                            'event-row-is-exception': row.event && row.event.event === '$exception',
+                            'event-row-date-separator': row.date_break,
+                            'event-row-new': row.new_events,
+                        })
+                    }}
+                    expandable={
+                        showRowExpanders
+                            ? {
+                                  expandedRowRender: function renderExpand({ event }) {
+                                      return event && <EventDetails event={event} />
+                                  },
+                                  rowExpandable: ({ event, date_break, new_events }) =>
+                                      date_break || new_events ? -1 : !!event,
+                                  noIndent: true,
+                              }
+                            : undefined
+                    }
+                />
+                {hasNext || isLoadingNext ? (
+                    <LemonButton
+                        type="primary"
+                        onClick={fetchNextEvents}
+                        loading={isLoadingNext}
+                        className="my-8 mx-auto"
+                    >
+                        Load more events
+                    </LemonButton>
+                ) : null}
+            </div>
+            <SessionPlayerDrawer />
```

Comment:
Note for later - I think we should refactor this component to work like the PersonsModal, so that the logic actually mounts the modal via a portal. Not for now though 👍